timed/reports/views.py

```diff
@@ -129,7 +132,10 @@ def get_queryset(self):
         queryset = Report.objects.all()
         queryset = queryset.values("task")
         queryset = queryset.annotate(duration=Sum("duration"))
-        queryset = queryset.annotate(pk=F("task"))
+        queryset = queryset.annotate(
+            pk=F("task"),
+            most_recent_remaining_effort=F("task__most_recent_remaining_effort"),
+        )
```

Comment:
Do we really need this here?

Or maybe we should change this queryset to start from `Task` instead. This would make tons of more sense and we'd avoid a lot of hacky stuff:

```python
queryset = Task.objects.all().annotate(duration=Sum('reports__duration'))
```

Reply:
Definitely. This change will come with the next PR. I'm already working on it.
This is just a temporary solution so we can use it for frontend adjustments/development.